tasks/build_tags.py

```diff
@@ -34,6 +34,7 @@
     "netcgo",  # Force the use of the CGO resolver. This will also have the effect of making the binary non-static
     "npm",
     "oracle",
+    "oracle-dbm",
```

Comment:
This build tag is not mentioned anywhere. Is it intentional?

Reply:
I removed them.